cmd/keytransparency-sequencer/main.go

```diff
@@ -140,12 +140,7 @@ func main() {
 		glog.Exitf("Failed to create directory storage object: %v", err)
 	}
 
-	creds, err := credentials.NewServerTLSFromFile(*certFile, *keyFile)
```

Comment:
Should `certFile` and `keyFile` be marked as deprecated?

Reply:
We still use `certFile` and `keyFile` - they just get used as part of `http.ServeTLS` rather than the go server itself.